src/js/_enqueues/lib/comment-reply.js

```diff
@@ -187,8 +189,9 @@ window.addComment = ( function( window ) {
 		var replyLink = this,
 			commId    = getDataAttribute( replyLink, 'belowelement'),
 			parentId  = getDataAttribute( replyLink, 'commentid' ),
-			respondId = getDataAttribute( replyLink, 'respondelement'),
-			postId    = getDataAttribute( replyLink, 'postid'),
+			respondId = getDataAttribute( replyLink, 'respondelement' ),
+			postId    = getDataAttribute( replyLink, 'postid' ),
+			replyTo   = getDataAttribute( replyLink, 'replyto' ) || 'Leave a Reply',
```

Comment:
:1234: i18n, the default will probably need to come in via PHP as the JS library isn't available on the front end and enqueueing it as a dependency of the comment reply script seems a little heavy handed.

```suggestion
			replyTo   = getDataAttribute( replyLink, 'replyto' ) || __( 'Leave a Reply' ),
```

Reply:
I've updated the PR to make use of the default heading text instead of hardcoded strings. I had to add some extra markup to the heading in order to do it cleanly, so I was also able to remove the `innerHTML` being used elsewhere.